### ISSUE TITLE
Filtering child nodes 

### ISSUE THREAD

**Issue by mundodojava** [wix/angular-tree-control#142]
Is it possible to filter not only siblings nodes but also child nodes?

**Comment by Damore** [comment]
Yes! I was reading the documentation and it is only possible to filter the siblings: "filter-expression : value for ng-repeat to use for filtering the sibling nodes"

It would be very helpful to have this functionality in the component.

**Comment by Damore** [comment]
Actually, the problem is that angular-tree-control Filtering does not work as expected on Angular 1.3.6. I just changed the version of my project to 1.3.16 and now the filtering is working fine!

**Comment by neoms21** [comment]
I'm using angular 1.4.1 but on filtering it doesn't expand all the nodes with matching filter (nodes are filtered fine just not expanded) although in example page it does it fine. I can't find anything which I'm missing.
